javascript/answers/ayushk12-q22.html

```diff
@@ -0,0 +1,81 @@
+<html>
+
+<head>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { dataLayer.push(arguments); }
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+    </script>
+
+    <title>
+        JavaScript | InterQues | Coding Ninjas
+    </title>
+
+    <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png" />
+
+    <meta name="description" content="What is the value of !'bang'?">
```

Comment:
content in meta tag should be question title.

Reply:
changes done